test/Benchmarks/Metrics/MetricsBenchmarks.cs

```diff
@@ -20,7 +20,6 @@
 using System.Diagnostics.Metrics;
 using BenchmarkDotNet.Attributes;
 using OpenTelemetry;
-using OpenTelemetry.Exporter;
```

Comment:
Do we know why these "unused using statements" are not discovered by the CI?

Reply:
I think /test folder is not covered.